frontend/src/components/BookingForm.js

```diff
@@ -34,6 +39,10 @@ const BookingForm = () => {
     setDateRange(getDateRangeArray())
   }, [selectedCheckInDate, selectedCheckOutDate])
 
+  useEffect(() => {
+    setUserData(userMetadata)
+  }, [user, userMetadata])
```

Comment:
```suggestion
```
We should just rename `userMetadata` to be `userData` here.

Reply:
Yea, probably. I initially thought that user was the actual user data itself.

Reply:
Yeah this is an unfortunate side-effect of the upstream hook we're using. I considered merging the two objects to be one, but settled for having it as a "metadata" part. Perhaps we can re-look that if it's still confusing.

Reply:
Yeah I was thinking this too. I only used that useEffect so that there was a reason to use the "user" variable since ESLint would get mad about unused variables. My smooth brain didn't know I could've just done `[, userData] = useAuthenticatedUser()` to destructure without having a user variable